README.md

```diff
@@ -43,14 +43,14 @@ version: '3'
 services:
   service_discovery:
     restart: unless-stopped
-    image: ghcr.io/eclipse-researchlabs/smartclide/service-discovery:2022-04-04
+    image: ghcr.io/eclipse-opensmartclide/smartclide/service-discovery:2022-04-04
```

Comment:
service-discovery:2022-04-04  -> does not exist

Reply:
@dabm-git should be fixed in latest commit